mlflow/utils/oss_registry_utils.py

```diff
@@ -25,5 +26,4 @@ def get_oss_host_creds(server_uri=None):
 
     if parsed_uri.path == _DATABRICKS_UNITY_CATALOG_SCHEME:
         return get_databricks_host_creds(parsed_uri.path)
-
-    return MlflowHostCreds(host=parsed_uri.path)
+    return MlflowHostCreds(host=parsed_uri.path, token=MLFLOW_UC_OSS_TOKEN.get())
```

Comment:
Do you want to have some error handling logic if the token env var is unset (prompt the user to ensure that it's configured)?

Reply:
The get will return None if it isn't set.  I think the AUTH error will be enough to indicate what the issue is.  I think it's also possible to configure UC OSS to not need auth, so I believe it is entirely possible for this to not be set.

Reply:
sounds reasonable to me. I think with some docs warnings for the connection ("remember to set this ENV VAR if you have auth configured for your UC OSS server!" sort of thing), we'll be good :)